goon/browserassets/js/browserOutput.js

```diff
@@ -113,7 +113,18 @@ function highlightTerms(el) {
 			highlightTerms(el.children[h]);
 		}
 	}
-	if (el.childNodes.length > 0 && typeof el.childNodes[0].data !== 'undefined' && el.childNodes[0].data.length > 0) { //If element actually has text
+
+	var hasTextNode = false;
+	for (var node = 0; node < el.childNodes.length; node++)
```

Comment:
Performance reasons.

If we have text nodes the below needs to be done, but if we don't it'd be wasteful to do so.